Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.py

```diff
@@ -0,0 +1,29 @@
+from CommonServerPython import *
+
+
+def main():
+    feed_url_to_config = {
+        'https://sslbl.abuse.ch/blacklist/sslipblacklist.csv': {
+            'fieldnames': ['date_created', 'value', 'port'],
+            'indicator_type': FeedIndicatorType.IP
+        },
+        'https://sslbl.abuse.ch/blacklist/sslipblacklist_aggressive.csv': {
+            'fieldnames': ['date_created', 'value', 'port'],
+            'indicator_type': FeedIndicatorType.IP
+        }
+    }
+
```

Comment:
We'll need to add a mapping to this here, correct?

Reply:
right